include/lsst/afw/table/detail/SchemaMapperImpl.h

```diff
@@ -78,7 +78,8 @@ class SchemaMapperImpl {
         }
 
         /// @brief Construct the wrappper.
-        explicit VisitorWrapper(F func) : _func(func) {}
+        template <typename T>
+        explicit VisitorWrapper(T func) : _func(std::forward<T>(func)) {}
```

Comment:
Yes, this should be `T&&` I just overlooked it. Of course the internal forwarding still works because it isn't used after but that is not really the point. Will fix it.